tests/queries/0_stateless/03271_ghdata_object_to_json_alter.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
-# Tags: no-fasttest, long
+# Tags: no-fasttest, long, no-s3-storage
+# ^ no-s3-storage: too slow
```

Comment:
It fails too often with s3 with timeout. Probably because of some random settings, but I couldn't find which exactly. I will try to play with settings later, for now let's disable test with s3